index.js

```diff
@@ -677,6 +677,7 @@ app.post(
           data.reminderTimer,
           data.fallbackTimer,
           newAlertApiKey,
+          data.isActive === 'true',
```

Comment:
ah, I see, this is for new locations. So I'll punt the question to design, whats the thought with defaulting to active? What about the use case of say, queuing up a location?

Reply:
This is just changing the string `data.isActive` into a real JS boolean value from the UI form.

Reply:
ah ok, I finally understood what this actually does, it supplies a boolean argument (i.e. "true" in a javascript sense if the string is === to 'true' and "false" as a javascript boolean otherwise). Got it.